plugin/src/index.ts

```diff
@@ -148,7 +148,7 @@ const plugin: NetlifyPlugin = {
 
     const middlewareManifest = await loadMiddlewareManifest(netlifyConfig)
 
-    if (!process.env.NEXT_USE_NETLIFY_EDGE && middlewareManifest.sortedMiddleware.length !== 0) {
+    if (!process.env.NEXT_USE_NETLIFY_EDGE && middlewareManifest?.sortedMiddleware?.length) {
```

Comment:
[dust] Consider being explicit about the length instead of coercing to boolean.

```suggestion
    if (!process.env.NEXT_USE_NETLIFY_EDGE && middlewareManifest?.sortedMiddleware?.length > 0) {
```

Reply:
That won't work - if middlewareManifest does not exist, then it resolves to undefined. `undefined > 0` will error.